frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditSoftwareModal/EditSoftwareModal.tsx

```diff
@@ -386,7 +411,7 @@ const EditSoftwareModal = ({
           isIosOrIpadosApp={isIosOrIpadosApp}
         />
       )}
-      {!!pendingPackageUpdates.software && isUpdatingSoftware && (
+      {!!pendingPackageUpdates.software && showFileProgressModal && (
```

Comment:
`!!pendingPackageUpdates.software` already prevents the modal from showing if the software file is not updated